frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/ISimpleGraphTableProvider.java

```diff
@@ -6,7 +6,7 @@
 
 public interface ISimpleGraphTableProvider {
   @UIEffect
-  BaseGraphTable deriveInstance(IGitMacheteRepositorySnapshot macheteRepositorySnapshot);
+  BaseGraphTable deriveInstance(IGitMacheteRepositorySnapshot macheteRepositorySnapshot, boolean toggleListingCommits);
```

Comment:
Hmm why this change is on this PR? :thinking:

Reply:
unrel, just fixing help dialog

Reply:
as this one is gonna be 0.7.1 we may want to ship it on the separate pr for 0.7.0

Reply:
Ok, I can

Reply:
Done